hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/RESTClientPhoneHomeTest.java

```diff
@@ -60,20 +61,22 @@ public static void beforeClass() {
     }
 
     @Rule
-    public WireMockRule wireMockRule = new WireMockRule();
+    public WireMockRule wireMockRule = new WireMockRule(wireMockConfig().dynamicPort());
```

Comment:
It may also make ''RESTClientPhoneHomeTest.queueOperations'' (issue #20044) test not fail again (if that fail was related to some weird parallelization issue), so additional benefit :)